libminifi/src/core/yaml/YamlConfiguration.cpp

```diff
@@ -805,6 +807,37 @@ void YamlConfiguration::parsePropertiesNodeYaml(const YAML::Node& propertiesNode
   validateComponentProperties(processor, component_name, yaml_section);
 }
 
+void YamlConfiguration::parseFunnelsYaml(const YAML::Node& node, core::ProcessGroup* parent) {
+  if (!parent) {
+    logger_->log_error("parseFunnelsYaml: no parent group was provided");
+    return;
+  }
+  if (!node || !node.IsSequence()) {
+    return;
+  }
+
+  for (YAML::const_iterator iter = node.begin(); iter != node.end(); ++iter) {
```

Comment:
Did you consider using a range-based for loop?

Reply:
Updated in eb396ae43b5dae3ec5c23f19725f5ac6d380bf78